src/xrpld/ledger/View.h

```diff
@@ -139,16 +152,67 @@ isFrozen(ReadView const& view, AccountID const& account, Issue const& issue)
 isFrozen(
     ReadView const& view,
     AccountID const& account,
-    MPTIssue const& mptIssue);
+    MPTIssue const& mptIssue,
+    int depth = 0);
 
 [[nodiscard]] inline bool
-isFrozen(ReadView const& view, AccountID const& account, Asset const& asset)
+isFrozen(
+    ReadView const& view,
+    AccountID const& account,
+    Asset const& asset,
+    int depth = 0)
 {
     return std::visit(
-        [&](auto const& issue) { return isFrozen(view, account, issue); },
+        [&](auto const& issue) {
+            return isFrozen(view, account, issue, depth);
+        },
         asset.value());
 }
 
+[[nodiscard]] bool
+isAnyFrozen(
+    ReadView const& view,
+    AccountID const& account1,
+    AccountID const& account2,
+    MPTIssue const& mptIssue,
+    int depth = 0);
+
+/*
```

Comment:
If there is no use case then should just delete the commented out block.

Reply:
this was actually un-commented.